examples/huggingface/pytorch/code-generation/quantization/run_generation.py

```diff
@@ -93,57 +94,68 @@
     ],
 )
 parser.add_argument(
-    "--woq_scale_dtype",
+    "--scale_dtype",
     type=str,
     default="fp32",
     choices=["fp32", "fp8"],
 )
-parser.add_argument("--woq_group_size", type=int, default=32)
-parser.add_argument("--woq_scheme", default="sym")
+parser.add_argument(
+    "--compute_dtype",
+    type=str,
+    default="fp32",
+    choices=["fp32", "bf16", "int8"],
+)
+parser.add_argument("--group_size", type=int, default=32)
+parser.add_argument("--scheme", default="sym")
 # ============GPTQ configs==============
 parser.add_argument(
-    "--gptq_actorder",
+    "--desc_act",
     action="store_true",
     help="Whether to apply the activation order GPTQ heuristic.",
 )
 parser.add_argument(
-    "--gptq_percdamp",
+    "--damp_percent",
     type=float,
     default=0.01,
     help="Percent of the average Hessian diagonal to use for dampening.",
 )
 parser.add_argument(
-    "--gptq_block_size",
+    "--blocksize",

```

Comment:
HF is group_size?

Reply:
no, it is only for gptq needed. we have another args named "group_size" corresponding to HF "group_size"

Reply:
what's the difference between group size and block size?

Reply:
blocksize, use to set GPTQ sub weight matrix size
group_size, use to divide weight to calculate scale